packages/client/components/ActivityLibrary/ActivityDetails.tsx

```diff
@@ -144,11 +196,18 @@ const ActivityDetails = (props: Props) => {
 
   const [isEditing, setIsEditing] = useState(false)
 
+  useEffect(
+    () =>
+      selectedTemplate &&
+      setActiveTemplate(atmosphere, selectedTemplate.teamId, templateId, selectedTemplate.type),
```

Comment:
There's some logic around dimensions + scale modifications that assume an active template in client-side global relay state. We should consider refactoring this in the long term, but simply setting the active template when the template changes is a reasonable quick-and-dirty solution in the near-term.